internal/services/appservice/helpers/service_plan.go

```diff
@@ -35,7 +35,7 @@ var sharedSkus = []string{
 }
 
 var consumptionSkus = []string{
-	"PC2", "PC3", "PC4", "Y1",
+	"Y1",
```

Comment:
actually do these also need to be removed from the docs?

Reply:
I got ahead of myself and did that in a previous PR 🙈